api/src/redis/use-redis.ts

```diff
@@ -0,0 +1,25 @@
+import { Redis } from 'ioredis';
+import { createRedis } from './create-redis.js';
+
+/**
+ * Memoization cache for useRedis
+ *
+ * @see {@link useRedis}
+ */
+export const _cache: { redis: Redis | undefined } = {
+	redis: undefined,
+};
```

Comment:
Is there a way to test it without exposing this internal value?

Reply:
Not really! Unless we export a function that clears the memoized state, which is effectively doing the same thing